buildconfig/download_win_prebuilt.py

```
@@ -106,8 +106,8 @@ def get_urls(x86=True, x64=True):
         'ea302368159ca64056b2519b3a624525f625220e',
         ],
         [
-        'https://github.com/libsdl-org/SDL_mixer/releases/download/prerelease-3.1.2/SDL3_mixer-devel-3.1.2-VC.zip',
-        '055992eb135197e74644bea65fdc8f8f0838b4ed'
+        'https://github.com/libsdl-org/SDL_mixer/releases/download/release-3.2.0/SDL3_mixer-devel-3.2.0-VC.zip',
```

Flake8 (7.3.0)

[error] continuation line missing indentation or outdented

(E122)